hadoop-ozone/dist/src/shell/ozone/ozone

```diff
@@ -187,6 +187,7 @@ function ozonecmd_case
     ;;
     fs)
       OZONE_CLASSNAME=org.apache.hadoop.fs.ozone.OzoneFsShell
+      OZONE_CLIENT_OPTS="${OZONE_CLIENT_OPTS} -Dhadoop.log.file=ozone-shell.log -Dlog4j.configuration=file:${ozone_shell_log4j}"
```

Comment:
I would suggest adding it in a different order, to avoid the problem HDDS-7153 is fixing for `ozone sh`.

```suggestion
      OZONE_CLIENT_OPTS="-Dhadoop.log.file=ozone-shell.log -Dlog4j.configuration=file:${ozone_shell_log4j} ${OZONE_CLIENT_OPTS}"
```

Reply:
Understood will do !!